README.rst

```diff
@@ -53,8 +53,24 @@ From GitLab:
    $ pip install git+https://gitlab.com/python-gitlab/python-gitlab.git
 
 
-Using the docker image
-----------------------
+Using the docker images
+-----------------------
+
+``python-gitlab`` provides Docker images in two flavors, based on the ``-alpine`` and ``-slim``
+python `base images <https://hub.docker.com/_/python>`__. The default tag is alpine, but you
+can explicitly use the alias (see below).
+
+The alpine image is smaller, but you may want to use the Debian-based ``-slim`` tag if you are
```

Comment:
That is confusing

Reply:
We have a smaller image, which is not called `slim` and the bigger image is called `slim` 🤔

Reply:
Yes, I was also pondering this in the issue https://github.com/python-gitlab/python-gitlab/issues/1852#issuecomment-1017393648 :( will try to come up with a better wording

Reply:
It's common to have `:alpine` and `:slim` with alpine being smaller, so maybe I can just add that `latest` is the same as `alpine` for backward compatibility?

Reply:
@max-wittig or I could simple move `:slim` to `:slim-bullseye` and then it's clear that "this is a slim debian image"?

https://github.com/docker-library/python/blob/d7fa897e3d1c57cf144b6253ffb08a11b7a7511c/3.10/bullseye/slim/Dockerfile

Reply:
@max-wittig switched to `slim-bullseye` to be more explicit about the base image.

Reply:
Looking at this again I don't really like it as people will get stuck on old images then, I'll take another look

Reply:
So now we just follow the python flavor tags, I've documented that. And we can upgrade the debian dist every few years manually.